dpos/manager/proposaldispatcher.go

```diff
@@ -463,6 +463,7 @@ func (p *ProposalDispatcher) IsCRCBadNetWork() bool {
 }
 
 func (p *ProposalDispatcher) IsViewChangedTimeOut() bool {
+	return false
```

Comment:
is return false Temporary? If so, maybe need add todo and If not del no use code .

Reply:
This change has been recorded in trello. It is a temp change to fix bug of main net.